core/src/main/java/com/linecorp/armeria/internal/client/SessionProtocolNegotiationCache.java

```diff
@@ -14,7 +14,7 @@
  * under the License.
  */
 
-package com.linecorp.armeria.client;
+package com.linecorp.armeria.internal.client;
```

Comment:
This class is actually meant for use by users as well. e.g. A user might want to check if a certain host has no support for H2.

Reply:
Ah I see, Let me revert this.